php/php.dbgp/src/org/netbeans/modules/php/dbgp/ui/DbgpExceptionBreakpointPanel.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.php.dbgp.ui;
+
+import javax.swing.JComponent;
+import javax.swing.JEditorPane;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import org.netbeans.api.debugger.DebuggerManager;
+import org.netbeans.editor.Utilities;
+import org.netbeans.modules.php.dbgp.breakpoints.ExceptionBreakpoint;
+import org.netbeans.modules.php.dbgp.ui.completion.ExceptionClassNbDebugEditorKit;
+import org.netbeans.spi.debugger.ui.Controller;
+import org.openide.DialogDisplayer;
+import org.openide.NotifyDescriptor;
+import org.openide.util.NbBundle;
+import org.openide.util.Pair;
+
+public class DbgpExceptionBreakpointPanel extends JPanel implements Controller {
+    private static final long serialVersionUID = 4967178020173186468L;
+    private static final String EMPTY_EXCEPTION_NAME = "LBL_EmptyExceptionName"; // NOI18N
+
+    private final JEditorPane epClassName;
+    private final JScrollPane spClassName;
+
+    public DbgpExceptionBreakpointPanel() {
+        initComponents();
+        Pair<JScrollPane, JEditorPane> editorCC = addClassNameEditorCC(ExceptionClassNbDebugEditorKit.MIME_TYPE, jPanel2, "", "");
+        spClassName = editorCC.first();
+        epClassName = editorCC.second();
+        epClassName.getAccessibleContext().setAccessibleName("pane");
+        epClassName.getAccessibleContext().setAccessibleDescription("pane");
```

Comment:
`// NOI18N`

Reply:
Well, these are not friendly accessible descriptions, right...?

Reply:
Please, improve A11Y description of these elements (imagine, that you cannot see them - how would you describe them?), thanks.